os/net/lwip/sys/arch/sys_arch.c

```diff
@@ -31,7 +31,7 @@
 #include <tinyara/cancelpt.h>
 #include <tinyara/kthread.h>
 #include <sys/types.h>
-
+#include <semaphore.h>
```

Comment:
Why include the header file here, if the rest of the file does not use it? Isn't it better to include files where they are used directly?

Reply:
will remove this line